backend/btrixcloud/models.py

```diff
@@ -977,16 +988,90 @@ class OrgQuotas(BaseModel):
 
 
 # ============================================================================
-class OrgCreate(BaseModel):
-    """Create a new org"""
+class SubscriptionCreate(BaseModel):
```

Comment:
Might be a good idea to add the oid to subscription events as well (even if we have to add it after the fact) so that we can filter per-org.

Reply:
Also done! Added to the subscription events and output models for the list endpoint

Reply:
I know we could get `subId` from the org then filter on that but adding oid directly will save users a step. Can revert if you think it's cleaner the other way.